provider/pkg/gen/python-templates/yaml/yaml.tmpl

```diff
@@ -164,7 +164,7 @@ class ConfigGroup(pulumi.ComponentResource):
 
         self.resources = pulumi.Output.from_input({})
 
-        _files: List[str] = []
+        _files: Sequence[str] = []
```

Comment:
I think we may need to keep this `List[str]` (and keep the `List` import) since it is modifying the list via `.append` below.